ui/src/components/Configs/UpsertConfig.tsx

```diff
@@ -24,13 +26,24 @@ type Props = {
   onClose: () => void;
 };
 
+const useStyles = makeStyles((theme: Theme) => createStyles({
+  emptyBox: {
+    height: theme.spacing(5),
+  },
+  textField: {
+    marginRight: theme.spacing(1),
+    marginTop: theme.spacing(1),
+  },
+}));
+
 export const UpsertConfig = ({ config, open, onClose }: Props): ReactElement => {
 
   const { runConfig, setRunConfig } = useRunConfig();
   const [newVar, setNewVar] = useState<string>('');
   const [newValue, setNewValue] = useState<string>('');
   const [configName, setConfigName] = useState<string>(config?.name || '');
-  const [newConfig, setNewConfig] = useState<RunConfig>(config || { name: '', vars: [] } as RunConfig);
+  const [newConfig, setNewConfig] = useState<RunConfig>(config || { name: '',id: uuid(), vars: [] } as RunConfig);
```

Comment:
nit:
`{ name: '', id: uuid(), vars: [] }` 
should we maybe set up automatic formatting/linting in this repo?